packages/zoltan2/src/algorithms/partition/Zoltan2_AlgMultiJagged.hpp

```diff
@@ -54,7037 +54,8852 @@
 #include <Zoltan2_Parameters.hpp>
 #include <Zoltan2_Algorithm.hpp>
 #include <Zoltan2_IntegerRangeList.hpp>
-#include <Teuchos_StandardParameterEntryValidators.hpp>
-
+#include <Zoltan2_CoordinatePartitioningGraph.hpp>
+#include <Zoltan2_Util.hpp>
 #include <Tpetra_Distributor.hpp>
+#include <Teuchos_StandardParameterEntryValidators.hpp>
 #include <Teuchos_ParameterList.hpp>
-#include <Zoltan2_CoordinatePartitioningGraph.hpp>
-#include <new>          // ::operator new[]
+
 #include <algorithm>    // std::sort
-#include <Zoltan2_Util.hpp>
 #include <vector>
-
-#if defined(__cplusplus) && __cplusplus >= 201103L
 #include <unordered_map>
-#else
-#include <Teuchos_Hashtable.hpp>
-#endif // C++11 is enabled
 
 #ifdef ZOLTAN2_USEZOLTANCOMM
 #ifdef HAVE_ZOLTAN2_MPI
-#define ENABLE_ZOLTAN_MIGRATION
+#define ZOLTAN2_MJ_ENABLE_ZOLTAN_MIGRATION
 #include "zoltan_comm_cpp.h"
 #include "zoltan_types.h" // for error codes
 #endif
 #endif
 
-#ifdef HAVE_ZOLTAN2_OMP
-#include <omp.h>
-#endif
-
-#define LEAST_SIGNIFICANCE 0.0001
-#define SIGNIFICANCE_MUL 1000
+#define ZOLTAN2_MJ_LEAST_SIGNIFICANCE 0.0001
+#define ZOLTAN2_MJ_SIGNIFICANCE_MUL 1000
 
 //if the (last dimension reduce all count) x the mpi world size
 //estimated to be bigger than this number then migration will be forced
 //in earlier iterations.
-#define FUTURE_REDUCEALL_CUTOFF 1500000
+#define ZOLTAN2_MJ_FUTURE_REDUCEALL_CUTOFF 1500000
+
 //if parts right before last dimension are estimated to have less than
 //MIN_WORK_LAST_DIM many coords, migration will be forced in earlier iterations.
-#define MIN_WORK_LAST_DIM 1000
-
-
-
+#define ZOLTAN2_MJ_MIN_WORK_LAST_DIM 1000
```

Comment:
This is better than before.  Consider `constexpr` constants instead of macros in future revisions.

Reply:
Switched to constexpr.